src/components/EditorSideBar/Blueprints.tsx

```diff
@@ -7,6 +7,7 @@ import { setActiveCourt } from "@/store/reducer/courtSpecDataSlice";
 import { mockTileData } from "../MockData/MockTileData";
 import { useStoreSelector } from "@/store/hooks";
 import { setCartDisplayState } from "@/store/reducer/cartControlSlice";
+import { reset } from "@/store/reducer/zoomCourtSlice";
```

Comment:
The name of this reset can be more meaningful. Cos it is not inside of any zoom related component, devs may not understand what "reset" for.

Reply:
> The name of this reset can be more meaningful. Cos it is not inside of any zoom related component, devs may not understand what "reset" for.

reset has been renamed as resetZoom